### ISSUE TITLE
Installer fails when keyserver is not reachable

### ISSUE THREAD

**Issue by jrohrauer** [grokability/snipe-it#15413]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

When installing Snipe-It through the installer script on a vanilla machine, it fails when the firewall blocks access to the keyserver (it usese a non-standard port 11371). The error message looks like this:

**/tmp/apt-key-gpghome.Dy3dgR5VNF/gpg.1.sh --keyserver keyserver.ubuntu.com --recv-keys 4F4EA0AAE5267A6C
gpg: keyserver receive failed: No keyserver available**

### Reproduction steps

1. Downloaded the script
2. Run the script
3.
...

### Expected behavior

I expect the script to run without errors :)

### Snipe-IT Version

7.0.11

### Operating System

Ubuntu 22.04

### Web Server

does not apply

### PHP Version

does not apply

### Additional context

The solution is quite easy. The install.sh script downloads the actual installer (at the time of writing, it is https://raw.githubusercontent.com/snipe/snipe-it/master/snipeit.sh). Line 580 and 624 both contain the following:
**sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 4F4EA0AAE5267A6C**

Replace ist with the following:
**gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 4F4EA0AAE5267A6C**

and run the script.

**Comment by snipe** [comment]
I'm not sure there's anything we can do about that. You'd need to poke a hole in your firewall to reach keyserver.ubuntu.com

**Comment by jrohrauer** [comment]
That is true. But for all the folks who can't, or have a grumpy firewall admin this workaround would help :)

As an alternative you could implement this change directly into your script. This would utilize port 80, which works most of the time.

________________________________
Von: snipe ***@***.***>
Gesendet: Donnerstag, 29. August 2024 17:33
An: snipe/snipe-it
Cc: jrohrauer; Author
Betreff: Re: [snipe/snipe-it] Installer fails when keyserver is not reachable (Issue #15413)


I'm not sure there's anything we can do about that. You'd need to poke a hole in your firewall to reach keyserver.ubuntu.com

**Comment by snipe** [comment]
This seems out of scope though. Closing this for now. Open a. new issue if you feel it’s still something that needs addressing.
